src/store-api/src/region_engine.rs

```diff
@@ -164,18 +164,19 @@ pub struct ScannerProperties {
     /// The first dim vector's length represents the output partition number. The second
     /// dim is ranges within one partition.
     pub partitions: Vec<Vec<PartitionRange>>,
+
+    pub append_mode: bool,
+
+    pub total_rows: usize,
```

Comment:
nitpick: Document the relationship between these two fields.

Reply:
fixed in 92e5295